streams/src/test/java/org/apache/kafka/streams/processor/internals/AbstractProcessorContextTest.java

```diff
@@ -179,6 +179,10 @@ public void appConfigsShouldReturnUnrecognizedValues() {
             equalTo("user-supplied-value"));
     }
 
+    @Test(expected = UnsupportedOperationException.class)
+    public void shouldThrowOnCurrentStreamTime() {
```

Comment:
This test seems not to be useful, as it tests the `TestProcessorContext` from below.